demos/fab.scss

```diff
@@ -26,3 +26,32 @@
 .demo-fab-icon-size {
   @include mdc-fab-icon-size(36px);
 }
+
+.demo-figure-fab-extended {
+  width: 100%;
+  padding: 0;
+  margin: 1em 40px;
+  box-sizing: border-box;
+}
+
+.demo-fab-extended-fluid {
+  @include mdc-fab-extended-fluid();
+
+  width: 100%;
+}
+
+@media (max-width: 700px) {
+  .demo-figure-fab-extended {
+    padding: 0 16px;
+    margin: 0;
+  }
+
+  .demo-fab-extended-fluid-responsive {
+
+    @include mdc-fab-extended-fluid();
+
+    &.mdc-fab {
```

Comment:
This is overly specific. The rule should only be:

```css
.demo-fab-extended-fluid-responsive {
  @include mdc-fab-extended-fluid;
  margin: 16px 0;
}
```

Reply:
Added this to improve specificity. We can remove this extra specificity since I've moved all styles from html file to sass file.